app/models/opentofu_worker.rb

```diff
@@ -21,6 +23,10 @@ def worker_deployment_name
     @worker_deployment_name ||= "#{deployment_prefix}opentofu-runner"
   end
 
+  def container_port
+    6001
```

Comment:
Shouldn't this be 6000 to match the service? https://github.com/ManageIQ/manageiq-pods/pull/1108/files#diff-3e26d51b3b228e4cc4a9256f6b4a49c75730fb226bb062aa45ff405040313608R29

Reply:
This is how it is in MiqUiWorker. Need to see why? but with this changes container starting on 6000

Reply:
As discussed in chat, this is 3001 for UI worker because that pod also has an internal httpd process that runs on 3000, so we need this method to tells the rails runner what port to start on, since it's not 3000.

I'm not sure if this would be used at all by the opentofu runner unless there's code to tell the opentofu-runner container to run on that port.

Reply:
This container port sets environment variable `PORT` on the container. Opentofu runner express server reads from this env var.
I am in process of testing my changes on the server.. will check this behavior to confirm.